oval-schemas/windows-system-characteristics-schema.xsd

```diff
@@ -797,7 +797,10 @@
      <!-- =============================================================================== -->
      <xsd:element name="ntuser_item" substitutionGroup="oval-sc:item">
           <xsd:annotation>
-               <xsd:documentation>The windows ntuser_item specifies information that can be collected from a particular ntuser.dat file.</xsd:documentation>
+               <xsd:documentation>To ensure consistent results across OVAL interpreters should comply with the following:</xsd:documentation>
+			  <xsd:documentation>* Create a 'does not exist' item for each ntuser.dat file that does contain the required registry key.</xsd:documentation>
+			  <xsd:documentation>This requirement goes against normal item creation in OVAL, but is very useful in reporting to the end user which ntuser.dat files failed registry key existence checks when tests mandate 'all_exist'</xsd:documentation>
```

Comment:
I'd say it should be documented as "strongly suggested" rather than "required", since you can't write tests against items that don't exist (except that you can, of course, test that a particular item doesn't exist).

Reply:
The word 'required' is only related to a test looking for a required key, the overall update is just a strong recommendation with "should comply with'

To ensure consistent results across OVAL interpreters should comply with the following: